lemur/static/app/angular/notifications/notification/notification.js

```diff
@@ -42,25 +42,15 @@ angular.module('lemur')
       PluginService.getByType('notification').then(function (plugins) {
         $scope.plugins = plugins;
         _.each($scope.plugins, function (plugin) {
-          if (plugin.slug === $scope.notification.pluginName) {
-            plugin.pluginOptions = $scope.notification.notificationOptions;
+          if (plugin.slug === $scope.notification.plugin.slug) {
+            plugin.pluginOptions = $scope.notification.plugin.pluginOptions;
```

Comment:
These fields were using incorrect variable names, resulting in the if statement never being executed and the plugin selection list showing up as empty.